prusti-viper/src/encoder/specification_blocks.rs

```diff
@@ -0,0 +1,174 @@
+use std::collections::{BTreeMap, BTreeSet};
```

Comment:
Is this file copied from something similar for the VIR high encoding? Can it use this directly?

Reply:
Using it directly now.